parlai/core/worlds.py

```diff
@@ -622,7 +625,7 @@ def display(self):
         return s
 
     def __len__(self):
-        return math.ceil(sum(len(w) for w in self.worlds) / len(self.worlds))
+        return len(self.world)
```

Comment:
how did that change from worlds to world? -- what's going on here?

Reply:
The length of a batch world was here defined as the ceiling of the average length of all the subworlds. However, subworlds are just shared versions of the seed world (self.world), so they should all be the same. (They are all the same in all of our worlds.). However, if you don't actually set up the data in the subworlds, they break when they try to return the length, unless you have the seed world share the length with all of them. (This only now just came up because of the new batching system.)

Reply:
ok